build.sbt

```diff
@@ -100,7 +100,10 @@ lazy val core = project
       "eu.timepit" %% "refined" % "0.9.28" % Test,
       // CATs
       "net.jcazevedo" %% "moultingyaml" % "0.4.2" % Test,
-      "io.github.classgraph" % "classgraph" % "4.8.143" % Test
+      "io.github.classgraph" % "classgraph" % "4.8.143" % Test,
+      // Config
+      "com.typesafe" % "config" % "1.4.2",
+      "com.iheart" %% "ficus" % "1.5.2"
```

Comment:
Can we try to avoid adding an external dependency?

Reply:
I don't see a way, not using these libs will force us to write our own configuration library 🤔 or change the structure of sangria so that users supply a config object when using it but this is the complex way

Reply:
OK I'll try on myself then.

Reply:
gonna close this PR because there is another simple way to achieve this by overriding the MeasureComplexity class reduceField function at the application level https://github.com/sangria-graphql/sangria/blob/09f744992477f2310e508a6dc7786353a6d51699/modules/core/src/main/scala/sangria/execution/QueryReducer.scala#L77 Thanks @yanns  for the super idea.